mantidproject/mantid
# Indirect Data Reduction ISIS Calibration - Resolution Rebinning range bars

The Range bars in the Resolution plot that represent the Rebinning Low and High should now automatically update to the correct values and should be able to me manipulated within the correct range.

**To test:**
- Change default instrument to OSIRIS (View > Preferences > Mantid)
- Open ISIS Calibration (Interfaces > Indirect > Indirect Data Reduction > ISIS Calibration)
- Ensure you have access to the Data archive turned on
- Ensure your `Instrument/Analyser/Reflection` is `OSIRIS / graphite / 002`
- Type in the run number `121313`
- Check the Create RES File option
- Ensure the range bars appear on the plot in the positions in the below:
  ![image](https://cloud.githubusercontent.com/assets/13132128/15575446/d4bbfa6a-234a-11e6-9eb4-3b52a412c3ab.png)
- Try to change the range bars  by dragging them. Ensure this can be done and it updates the interface.
- Ensure the range bars are constrained to `+/- 0.245`
- Ensure that dragging the lower range bar over past the upper range bar switches their values in the Property table (e.g. so High is always more than Low)

Fixes #16411

[RELEASE NOTES](https://github.com/mantidproject/mantid/blob/e64b4c99fe61a4adf601aa6185e78409787f89cd/docs/source/release/v3.7.0/indirect_inelastic.rst#bugfixes)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
